userdocs/profiles.dev/docs/author-docs/local-helm-chart.md

```diff
@@ -0,0 +1,58 @@
+---
+sidebar_position: 3
+---
+
+# Local Helm Chart artifacts
+
+A "local" Helm Chart artifact means the chart itself, and any supporting material,
+is stored locally in the profile directory. On installation, these local manifests
+will be applied, rather than fetched down from a remote chart server.
```

Comment:
I'm fairly sure one of the oxford commas is not needed here. :D

Reply:
I feel like applied is a wrong wording here, because we don't do that. I would rather say, downloaded to a local folder for the user to apply... or something in that direction.

Reply:
> I'm fairly sure one of the oxford commas is not needed here.

oh yeh definitely

> I would rather say, downloaded to a local folder for the user to apply... or something in that direction.

👍